CVEs/CVE-2021-31712.json

```
@@ -0,0 +1,23 @@
+{
+    "CVE": "CVE-2021-31712",
+    "state": "PUBLISHED",
+    "repository": ["https://github.com/jpuri/react-draft-wysiwyg.git"],
+    "prePatch": {
+        "commit": "f59ee8419cdbd45aab3bdfdf1995f112b09bbb6a",
+        "weaknesses": [
+            {
+                "location": {
+                    "file": "src/decorators/Link/index.js",
+                    "line": [35, 59]
+                },
+                "explanation": "react-draft-wysiwyg (aka React Draft Wysiwyg) before 1.14.6 allows a javascript: URi in a Link Target of the link decorator in decorators/Link/index.js when a draft is shared across users, leading to XSS."
+            }
```

The `line` has to be a number, so you need to split out the weaknesses.   
The same thing for CVE-2021-23364. 
(That's why the CI checks are failing). 

```suggestion
            {
                "location": {
                    "file": "src/decorators/Link/index.js",
                    "line": 35
                },
                "explanation": "react-draft-wysiwyg (aka React Draft Wysiwyg) before 1.14.6 allows a javascript: URi in a Link Target of the link decorator in decorators/Link/index.js when a draft is shared across users, leading to XSS."
            },
            {
                "location": {
                    "file": "src/decorators/Link/index.js",
                    "line": 59
                },
                "explanation": "react-draft-wysiwyg (aka React Draft Wysiwyg) before 1.14.6 allows a javascript: URi in a Link Target of the link decorator in decorators/Link/index.js when a draft is shared across users, leading to XSS."
            }
```

---

Sorry, I thought it could be an array.

Thanks!